### ISSUE TITLE
Starknet Scaffold Documentation

### ISSUE THREAD

**Issue by Darlington02** [horuslabsio/Starknet-Scaffold#82]
Amazing job we've done thus far, but we need a great documentation, people can reference as there's not so much we can cover in the Readme.

**This issue will require 8 persons, with great writing skills. The documentation will include the following sections:**

- [x] Introduction - @Immanuelolivia1 
- [x] Quickstart/installation - @Joy-Adah 
- [x] UI components - @Oshioke-Salaki 
- [x] A tour guide of the different features [Deployer, Burner, Faucet, Wikipedia, Converter] - @MarvyNwaokobia 
- [x] A breakdown of the scripts - @kateberryd 
- [x] Spinning up and using devnet - @od-hunter 
- [x] Starknet Address Book - @Gift-Naomi 
- [x] Contributing to Starknet Scaffold - @mimiprosper

**Comment by Immanuelolivia1** [comment]
i

**Comment by MarvyNwaokobia** [comment]
k

**Comment by BlackStarkGoku** [comment]
a

**Comment by vibenedict** [comment]
i will like to jump on this issue

**Comment by martinvibes** [comment]
i

**Comment by petersdt** [comment]
i

**Comment by kateberryd** [comment]
@Darlington02 Can I work on this

**Comment by od-hunter** [comment]
https://yakihonne.com/article/naddr1qq2457n9vdzhz5jtve2455z32en97uzlf3yhyq3qcst96d8s7sstz8wqjtnu34rkqh4wpn6yt9d48v472ja7qvl0ckasxpqqqp65w0w0hv4.  Here is a link to my work, i can take two topics

**Comment by Darlington02** [comment]
All assigned persons should please indicate on the telegram group so we can sync on the documentation. @Immanuelolivia1 you'd be leading the efforts.

**Comment by CollinsC1O** [comment]
Hi @Darlington02 I will love to work on this

**Comment by Immanuelolivia1** [comment]
Can I work on this too ?

**Comment by VictoriaAde** [comment]
Hi, I'm a technical writer.
I would like to work on this

**Comment by mimiprosper** [comment]
Am aa technical writer. I would like to work on this project

https://medium.com/@emma.onyedika.okeke

**Comment by kateberryd** [comment]
@Darlington02 I would like to work on this.

Here are articles I have written.
[](https://hackmd.io/rxW2rRART16Pgdc75-CA4Q)
[](https://hackmd.io/XD78VPGvTru2PryAM0BqAw)

**Comment by KeneePatel** [comment]
Can I be assigned to the docs for making Contributing.md ? I have done the setup for the project and will take inspiration from other CONTRIBUTING.md files of well-known Starknet projects.

**Comment by Oshioke-Salaki** [comment]
I'd love to be a part of this

**Comment by Jemiiah** [comment]
@Darlington02 i would love to work on this documentation and here is an article i have worked on before

**Comment by No-bodyq** [comment]
@Darlington02 can I get in on this

**Comment by martinvibes** [comment]
@Darlington02 i'd love to work on this

**Comment by od-hunter** [comment]
@Darlington02 I’d like to work on this too please

**Comment by Gift-Naomi** [comment]
@Darlington02 Can i also be assigned

Here is an article I've written
https://hackmd.io/5oB7fqBwTpS7_Sojc01Ggg

**Comment by petersdt** [comment]
hi @Darlington02 am interested in contributing to the article

**Comment by IjayAbby** [comment]
Hello @Darlington02 I don't have many personal articles but in my previous role as a dev I worked for a startup company for 2 years, where I was writing the code and documentation at the same time. In short, I have expertise simpilfying high-level code to human friendly interaction regardless of their expertize whether technical or non-technical. 
Recently, I have writing twitter threads explaining different blockchain concepts for anyone which have been engaging and I have been able to onboard more users in blockchain ecosystem.
My recent web3 article: https://medium.com/@inyangasi4/deploying-a-simple-nft-on-polygon-mumbai-and-listing-it-on-opensea-marketplace-330ab7eb8cb0
Would love to be given a chance to make my first contribution on ODHack. 
Thanks
